api/src/test/java/uk/gov/dhsc/htbhf/claimant/AbstractIntegrationTest.java

```diff
@@ -0,0 +1,53 @@
+package uk.gov.dhsc.htbhf.claimant;
+
+import org.assertj.core.api.AssertionsForInterfaceTypes;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.web.client.TestRestTemplate;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import uk.gov.dhsc.htbhf.claimant.controller.ErrorResponse;
+import uk.gov.dhsc.htbhf.claimant.entity.Address;
+import uk.gov.dhsc.htbhf.claimant.entity.Claimant;
+import uk.gov.dhsc.htbhf.claimant.model.AddressDTO;
+import uk.gov.dhsc.htbhf.claimant.model.ClaimantDTO;
+
+import java.net.URI;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static org.springframework.boot.test.context.SpringBootTest.WebEnvironment.RANDOM_PORT;
+
+@SpringBootTest(webEnvironment = RANDOM_PORT)
+public abstract class AbstractIntegrationTest {
```

Comment:
Something to consider: How about making this class static and using static imports on the test classes? Then you just need to move the restTemplate back into the test classes (where it is probably better situated anyway)

Reply:
Yes, true, this was a better fit when I originally did this, but I've moved more and more out so a static utility class is now a better fit.